src/current/v24.1/schema-design-schema.md

```diff
@@ -126,6 +126,10 @@ Create a user certificate for `abbey` as well:
 $ cockroach cert create-client abbey --certs-dir={certs-directory} --ca-key={my-safe-directory}/ca.key
 ~~~
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/misc/cert-auth-using-x509-subject.md %}
```

Comment:
We don't need to include this in every file that references `cockroach cert` (and I'll stop commenting on this)

Reply:
:-) 

ok, removed in this instance and I also reviewed the PR and removed it from a bunch of places that just happened to mention `cockroach cert`, including the various Java app pages etc

I left it on some more reference-y pages about cert generation and how auth works, etc. so folks who need it can hopefully find their way to it from there (and search / AI will get them there regardless I suppose)